scripts/postinstall.js

```diff
@@ -3,31 +3,32 @@ const colors = require('colors')
 // eslint-disable-next-line
 const fs = require('fs')
 
-const supportedCdkVersion = '"1.49.1"'
+const supportedCdkVersion = '1.49.1'
+const supportedCdkVersionString = `"${supportedCdkVersion}"`
```

Comment:
I'm wondering about the reason for this extra variable?
Have you tried removing the extra set of quotes, e.g.
`const supportedCdkVersion = '1.49.1'` 
(on the previous commit, not this one)

Reply:
done